docs/src/overview/kubernetes.md

```diff
@@ -44,3 +44,112 @@ Due to the reason we use ignition in our operating system images for userdata, w
 ### machine-controller-manager-provider-metal
 
 Worker nodes are managed through Gardener's [machine-controller-manager](https://github.com/gardener/machine-controller-manager) (MCM). The MCM allows out-of-tree provider implementation via sidecar, which is what we implemented in the [machine-controller-manager-provider-metal](https://github.com/metal-stack/machine-controller-manager-provider-metal) repository.
+
+
+## Gardener with metal-stack
```

Comment:
I think existing Gardener sections should go into this new chapter. Introductory sentence of this document should be made independent of Gardener and moved to here.